src/db/oracledb/addresses-dao.js

```diff
@@ -0,0 +1,38 @@
+import _ from 'lodash';
+
+import { serializeAddresses } from 'serializers/addresses-serializer';
+import { parseQuery } from 'utils/parse-query';
+import { getConnection } from './connection';
+import { contrib } from './contrib/contrib';
+
+/**
+ * Queries data source for raw address data and passes it to the serializer before returning
+ *
+ * @param {string} osuId OSU ID of person to select addresses from
+ * @param {object} query Query parameters passed in with request
+ * @returns {Promise<object>} Serialized address resource
+ */
+const getAddressesById = async (osuId, query) => {
+  const connection = await getConnection();
+  try {
+    const parsedQuery = parseQuery(query);
+    parsedQuery.osuId = osuId;
+    parsedQuery.addressType = parsedQuery.addressType ? parsedQuery.addressType : null;
+    parsedQuery.city = parsedQuery.city ? parsedQuery.city : null;
+    parsedQuery.county = parsedQuery.county ? parsedQuery.county : null;
+    parsedQuery.stateCode = parsedQuery.stateCode ? parsedQuery.stateCode : null;
+    parsedQuery.nationCode = parsedQuery.nationCode ? parsedQuery.nationCode : null;
```

Comment:
This logic is repeated for all of these properties, so I think it can be refactored.